tools/ExtensionDownloader.ps1

```diff
@@ -138,11 +138,15 @@ function Main
 
             $r = mkdir $itemDir;
     
-            if((Test-Path $cabFile) -eq $False )
+            # Always download to ensure we are verifying the correct latest file
+            if ((Test-Path $cabFile) -eq $True)
             {
-                Write-Host "Downloading cab:" $objectInfo.downloadLocation "to:" $itemDir;
-                Invoke-WebRequest -Uri $objectInfo.downloadLocation -OutFile $cabFile;
+                Write-Error "File: " $itemDir + " already exists. This is unexpected.";
```

Comment:
> +  [](start = 45, length = 3)

1. don't need the + sign... it will actually just get logged in the output.
2. you should log the $cabFile, not the directory.
3. you don't need the space after the ':' or before the " already" (see line 148 below)